source/getting-started/javascript.md

```diff
@@ -90,48 +95,269 @@ See also: [app authorization in the API reference](/reference/#authorizing-your-
+
+#### Create
+
+```javascript
+// If no id is set, one is generated;
+// If parentId is null, a "root" stream is created
+var streamData = {
+  name: 'A Stream',
+  id: 'a-stream-id',
+  parentId: 'valid-stream-id'
+};
+
+var stream = new pryv.Stream(connection, streamData);
+connection.streams.create(stream, function (err, streamCreated) { 
+  // ...
+});
+```
+
+#### Update
+
+```javascript
+var stream = {
+  id: 'a-stream-id',
+  // Here we update the name of the stream created above
+  name: 'An Updated Stream'
+}
+
+connection.streams.update(stream, function (err, streamUpdated) {
+  // ...
+});
+```
+
+#### Delete
+
+```javascript
+// Only the id of a stream is needed for removal
+connection.streams.delete(stream, function (err, streamDeleted) {
+  // ...
+}, mergeEventsWithParent);
+```
+
+### Manage accesses
+
+#### Retrieve
+
+```javascript
+connection.accesses.get(function (err, accessList) {
+  // ...
+});
+```
+
+#### Create
+
+```javascript
+var access = {
+  name: 'An Access',
+  permissions: [
+    {
+      streamId: 'valid-stream-id',
+      level: 'manage'
+    }
+  ]
 };
-connection.events.create(eventData, function (err, event) { 
+
+connection.accesses.create(access, function (err, accessCreated) { 
   // ...
 });
 ```
 
 #### Update
 
 ```javascript
-event.content = 'Updated content.';
-event.update(function (err, updatedEvent) {
+var access = {
+  id: 'an-access-id',
+  name: 'An Updated Access'
+  permissions: [
+    {
+      streamId: 'valid-stream-id',
+      level: 'contribute'
+    }
+  ]
+}
+
+connection.accesses.update(access, function (err, accessUpdated) {
   // ...
 });
 ```
 
 #### Delete
 
 ```javascript
-event.delete(function (err, trashedEventOrNull) {
+// Only the id of an access is needed for removal
+connection.accesses.delete(access, function (err, accessDeleted) {
+  // ...
+});
+```
+
+### Batch calls
+
+```javascript
+var methodsData = [
+  {
+    'method': 'streams.create',
+    'params': {
+      'id': 'a-new-stream',
+      'name': 'A New Stream'
+    }
+  },
+  {
+    'method': 'events.create',
+    'params': {
+      'streamId': 'a-new-stream',
+      'type': 'note/txt',
+      'content': 'This is a new event.'
+    }
+  },
+  {
+    'method': 'accesses.create',
+    'params': {
+      'name': 'A New Access',
+      'permissions': [
+        {
+          'streamId': 'a-new-stream',
+          'level': 'read'
+        }
+      ]
+    }
+  }
+];
+
+connection.batchCall(methodsData, function (err, results) {
+  //...
+});
+```
+
+### Monitor
+
+#### Setup Monitor
+```javascript
+var filter = new pryv.Filter({limit: 5});
+var monitor = connection.monitor(filter);
+
+// This whill look in cache before looking online, default is false
+monitor.useCacheForEventsGetAllAndCompare = false;
+// This whill optimize start up by prefecthing some events, default is 100
+monitor.ensureFullCache = false;
+// This will fetch all events on start up, default is true
+monitor.initWithPrefetch = 0;
+```
+
+#### Load
```

Comment:
More description of what Load, event/structure/filter change stand for
Either as inline comment or just before the code block.